src/mc_rbdyn/Robot.cpp

```diff
@@ -219,10 +219,7 @@ Robot::Robot(Robots & robots,
   for(auto & fs : forceSensors_)
   {
     bfs::path calib_file = bfs::path(module_.calib_dir) / std::string("calib_data." + fs.name());
-    if(bfs::exists(calib_file))
-    {
-      fs.loadCalibrator(calib_file.string(), mbc().gravity);
-    }
+    fs.loadCalibrator(calib_file.string(), -mbc().gravity);
```

Comment:
Hmm... this minus is a big change, is that desired?

Reply:
No! It popped up again :S
Sorry it must have been lurking around in one of the old commits in Kevin's branch. Good catch!